### PR TITLE
TINY-8391: Fix test relying on previous state

### PR DESCRIPTION
**Pull request by ltrouton**
Related Ticket: TINY-8391

Description of Changes:
* Fix test error that was exposed from chunking shift

Pre-checks:
* [X] ~~Changelog entry added~~
* [X] Tests have been added (if applicable)
* [X] Branch prefixed with `feature/` for new features (if applicable)

Review:
* [X] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

File: modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts [hunk 1]
```diff
@@ -97,6 +97,12 @@ describe('browser.tinymce.themes.silver.editor.color.ColorPickerSanityTest', ()
 
       it('TBA: Open dialog, pick a different color, click Cancel and assert color does not change', async () => {
```
Comment (lnewson) [inline modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts:98]: This needs updating to reflect the changes in the test
```suggestion
      it('TBA: Open dialog, pick black, reopen the dialog, pick a different color, click Cancel and assert color does not change', async () => {
```

**Review by lnewson — APPROVED** [review]

File: modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts [hunk 2]
```diff
@@ -97,6 +97,12 @@ describe('browser.tinymce.themes.silver.editor.color.ColorPickerSanityTest', ()
...
         const editor = hook.editor();
+        // Change color to black
+        await pOpenDialog(editor);
+        await pSetHexBlack(editor);
+        TinyUiActions.submitDialog(editor);
+        await pWaitForDialogClose(editor);
```
Comment (lnewson) [inline modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts:104]: Actually... thinking about this more, should just do a `setContent` call instead?

Comment (ltrouton) [reply modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts:104]: The way the test file is set up, it looks like it doesn't use what is present in the editor at all. I'll confirm that though

**Review by TheSpyder — APPROVED** [review]

File: modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts (hunk 2 above)
Comment (ltrouton) [reply modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts:104]: Looks like that is the case. I think I might just change the test message to more accurately point out what is happening. Thanks for pointing that out

**Pull request merged**